contributingGuides/STYLE.md

```diff
@@ -112,38 +112,9 @@ if (someCondition) {
 }
 ```
 
-## Object / Array Methods
-
-We have standardized on using [underscore.js](https://underscorejs.org/) methods for objects and collections instead of the native [Array instance methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#instance_methods). This is mostly to maintain consistency, but there are some type safety features and conveniences that underscore methods provide us e.g. the ability to iterate over an object and the lack of a `TypeError` thrown if a variable is `undefined`.
```

Comment:
I think we should update this to `standardized on using lodash or native method` instead of deleting it

Reply:
We have `you-dont-need-lodash-underscore`, so pretty sure we favor to use native object/array method. For example, instead of `lodashKeys`, we prefer `Object.keys`.

Reply:
Should we still add this information in here? Seems like there is a preference and posibility of using lodash so it makes sense to leave a guide here in terms of what is preferred